cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -33,6 +33,11 @@ import (
 const (
 	vulnCVETruncateLimit = 10
 	DefaultDepth         = 1
+	// trivyTimeout defines the value that will be passed to the trivy CLI via
+	// the `--timeout` flag. The value should be bigger than the check timeout
```

Comment:
Do not use backticks in doc comments. They are not rendered as pre-formatted text.

Reply:
Thanks for the suggestions. Both changes have been applied.